response.go

```diff
@@ -130,6 +130,41 @@ func (r *Response) Image(im image.Image) error {
 	return nil
 }
 
+// Attachment send image, audio, video and file attachment
+func (r *Response) Attachment(dataType, url string) error {
```

Comment:
We need `AttachmentType` const value unless implement the following method.

```go
func (m *Messenger) ImageAttachment
func (m *Messenger) AudioAttachment
func (m *Messenger) VideoAttachment
func (m *Messenger) FileAttachment
```
and 

```diff
m := SendStructuredMessage{
    Recipient: r.to,
    Message: StructuredMessageData{
        Attachment: StructuredMessageAttachment{
-           Type: dataType,
+           Type: FileAttachment,
            Payload: StructuredMessagePayload{
                Url: url,
            },
        },
    },
}
```